### PR TITLE
Adds output slots table editor

### PR DESCRIPTION
**Pull request by martinRenou**
Adds the output slots table editor for both the `DataSourceModelView` and `KPICalculatorModelView`. 
Also adds an `EvaluatorModelView` to prevent code duplication.

### CODE REVIEW

File: force_wfmanager/left_side_pane/evaluator_model_view.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    def update_model(self):
+        self.model.output_slot_names[self.index] = self.name
+
+
+output_slots_editor = TableEditor(
+    sortable=False,
+    configurable=False,
+    auto_size=False,
+    columns=[
+        ObjectColumn(name="type", label="Output", editable=False),
```
Comment (martinRenou) [inline force_wfmanager/left_side_pane/evaluator_model_view.py]: Should be label=type

File: force_wfmanager/left_side_pane/evaluator_model_view.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        ObjectColumn(name="name", label="Name", editable=True),
```
Comment (martinRenou) [inline force_wfmanager/left_side_pane/evaluator_model_view.py]: should be "Variable name". And add index column

File: force_wfmanager/left_side_pane/evaluator_model_view.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    )
+
+    def __init__(self, model, *args, **kwargs):
+        self.model = model
+        if isinstance(model, BaseDataSourceModel):
+            self._evaluator = model.factory.create_data_source()
+        elif isinstance(model, BaseKPICalculatorModel):
+            self._evaluator = model.factory.create_kpi_calculator()
+
+        self._update_output_slots_table()
```
Comment (martinRenou) [inline force_wfmanager/left_side_pane/evaluator_model_view.py]: Move this line under the `super` call

File: force_wfmanager/left_side_pane/evaluator_model_view.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        UItem(
+            "output_slots_representation",
+            editor=output_slots_editor,
+        ),
+        kind="subpanel",
+    )
+
+    def __init__(self, model, *args, **kwargs):
+        self.model = model
+        if isinstance(model, BaseDataSourceModel):
```
Comment (martinRenou) [inline force_wfmanager/left_side_pane/evaluator_model_view.py]: put this in the default method

File: force_wfmanager/left_side_pane/evaluator_model_view.py [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+        self._update_output_slots_table()
+
+    def _label_default(self):
+        return get_factory_name(self.model.factory)
+
+    def __evaluator_default(self):
+        if isinstance(self.model, BaseDataSourceModel):
+            return self.model.factory.create_data_source()
+        elif isinstance(self.model, BaseKPICalculatorModel):
+            return self.model.factory.create_kpi_calculator()
```
Comment (stefanoborini) [inline force_wfmanager/left_side_pane/evaluator_model_view.py]: else throw exception...

File: force_wfmanager/left_side_pane/evaluator_model_view.py [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+    @on_trait_change('model.changes_slots')
+    def _update_output_slots_table(self):
+        _, output_slots = self._evaluator.slots(self.model)
+
+        #: Initialize the output_slot_names in the model
+        self.model.output_slot_names = len(output_slots)*['']
+
+        # Initialize slot names representation for the tables
+        self.output_slots_representation = [
+            OutputSlot(type=output_slot.type, model=self.model, index=index)
```
Comment (stefanoborini) [inline force_wfmanager/left_side_pane/evaluator_model_view.py]: index, type, model

**Pull request merged**